job-descriptions/ux-designer.md

```diff
@@ -0,0 +1,63 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# UX Designer (San Francisco or remote)
+
+### About us
+
+Sourcegraph's mission is to enable every software developer to create products using the best technology. The innovations of the future will all rely on software, and we want to bring the future sooner by building the best tools that help developers answer questions about their code. You can learn more about our mission by reading [our master plan](https://sourcegraph.com/plan).
+
+Our values:
+
+- **Inclusivity**. To build a product that serves the needs of all developers, we need to have a diverse team. Sourcegraph is an equal opportunity workplace; we embrace diversity and welcome people from all backgrounds and communities.
+- **Openness**. We are an [open company](https://docs.sourcegraph.com/dev/open_source_open_company): our code, our product roadmap, and our company processes are public. This transparency helps us rapidly gather feedback from our customers so we can iterate, learn, and deliver the best product.
+- **Progress**. We make continuous progress on our product by [releasing a new version of Sourcegraph every month](https://docs.sourcegraph.com/dev/releases). Check out [our product roadmap](https://docs.sourcegraph.com/dev/roadmap) to see what we are working on next. We also hold [regular retrospectives](https://docs.sourcegraph.com/dev/retrospectives) to learn how we can work better as a team.
+
+If you are passionate about making the world better through software, come join us!
+
+### About the role
+
+We are searching for a creative, self-motivated UX designer who can brainstorm ideas and bring them to life. We highly value Product Design within our organization, as it impacts all aspects of Sourcegraph’s product. You'll have creative autonomy and a strong influence over the product’s development.
+
+You will:
+
+- Own the comprehensive user experience of the product.
+- Contribute to overall strategy and decision-making about Sourcegraph’s features.
+- Work closely and effectively communicate with our product manager and engineers to define, design, and improve Sourcegraph, ensuring a high-quality implementation.
+- Support multiple projects simultaneously while meeting tight deadlines.
+- Create designs that adhere to and extend the existing style guide.
+
+### Ideal candidates have
+
+- An understanding of and passion for what makes a great user experience.
+- A portfolio showcasing their skillset and years of experience designing digital products.
+- Experience (limited, or as a hobby, is fine) as a programmer to understand the product and user need, and to jump into our codebase and fix design (e.g. CSS) issues.
+- A knack for giving and receiving feedback, and explaining the reasoning behind their designs.
+- Experience using Figma, Sketch, or any other vector-based design tools to deliver product ideas from conception to mockups and finished renderings.
```

Comment:
Is this a qualification? It is hard to imagine a designer being successful without experience like this.